src/main/java/com/Zoko061602/SuperTic/compat/Thaumcraft.java

```diff
@@ -66,8 +69,13 @@ static void registerRecipes() {
             appleeasy = applehard;
             applehard = new ItemStack(TinkerArmor.diamondApple);
         }
+        @SuppressWarnings("unchecked")
+        List<Object> recipes = ThaumcraftApi.getCraftingRecipes();
         infusion = new InfusionModifierRecipe(new ItemStack(tools[0]));
-        for (int i = 1; !(i == tools.length); i++) new InfusionModifierRecipe(new ItemStack(tools[i]));
+        recipes.add(infusion);
+        for (int i = 1; !(i == tools.length); i++) {
+            recipes.add(new InfusionModifierRecipe(new ItemStack(tools[i])));
+        }
```

Comment:
```suggestion
        infusion = new InfusionModifierRecipe(new ItemStack(TinkerTools.pickaxe));
        recipes.add(infusion);
```

Not 100% sure this will work, but I am about 99% sure that registering one recipe will be enough to handle every single tool.

This will also allow the `tools` constant to be removed.

Reply:
it would certainly remove them from NEI, though its not like this works perfectly with NEI anyway